### DISCUSSION TITLE
Integration with fly.io

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [loxilb-io/loxilb#734] [Ideas]
They allow the use of dedicated anycast IPV4 address at a pretty low cost, pretty good for low traffic homelab users.
https://fly.io/docs/networking/services/#dedicated-ipv4

**Comment by TrekkieCoder** [comment]
It is a super interesting interesting project. It will be really good to integrate loxilb with fly.io

**Comment by TrekkieCoder** [comment]
@stevefan1999-personal  Is there some document how to integrate 3rs party LB or networking components  with fly.io  ?

**Reply by stevefan1999-personal** [reply]
I'm not sure, but we need to enable IPV4 address allocation first, that cost $2 monthly atm, but it should very well be in free quota if you use the smallest VM type to run the loxilb docker image
